state/my_lists/operations/myLists.js

```diff
@@ -0,0 +1,35 @@
+import { createLogic } from 'redux-logic';
+import { normalize, schema } from 'normalizr';
+
+import { FETCH_MY_LISTS } from '../types';
+import { isUserLoggedIn } from '../../session/selectors';
+import { fetchMyListsSuccess, fetchMyListsError } from '../actions';
+import { mergeListsList } from '../../data/actions';
+
+const myListsLogic = createLogic({
+  type: FETCH_MY_LISTS,
+
+  async process({ getState, httpClient }, dispatch, done) {
+    const state = getState();
+    const sessinId = isUserLoggedIn(state);
```

Comment:
It's better to use selector `getSessionId` which have name which describes returned value.

Reply:
Under the hood, getSessionId has the same logic as isUserLoggedIn. Isn't it?

Reply:
Yes, but as another developer who using this code, I expect to get boolean value from function called `isUserLoggedIn`, just true or false. This might be used as:

```
const getSessionId = (state) => state.session.sessionId;
const isUserLoggedIn = (state) => !!getSessionId(state);
```

Reply:
I understood. Fixed.